docs/teams.md

```diff
@@ -41,11 +41,12 @@ From the cli of your host: `docker-compose exec openconext tail -50  /var/log/te
 The docker container does not contain a functioning email service, and so sending an invite / creating a team does not actually send a mail.
 
 Therefore: to test with an actual member, you need to do the following:
-1. open the openconext docker cli
-2. go to mysql (`mysql` on the cli)
-3. select a uiid for one of the invited people: 
-   1. `use teams;`
-   2. `select invitation_uiid from invitations where id = ;` => fill in the id
-4. accept the invite by going to the following url: `https://teams.vm.openconext.org/invitation/accept/$uiid_of_the_invite` where you replace the last bit with the uiid found in the previous step.
-
-If you use the cli to connect to docker: `docker-compose exec openconext mysql -e 'use teams; select invitation_uiid from invitations where id = ;'` & so on.
+1. Find the invitation_uiid of the user you want to accept `docker-compose exec openconext mysql -e "select * from invitations \G" teams`
+When accepting a new invite a new private window needs to opened, otherwise the following error will show: "You are already a member of this Team."
```

Comment:
"needs to open" instead of "needs to be opened"

Reply:
nicely spotted aswell